client-src/elements/chromedash-header.js

```diff
@@ -164,10 +164,10 @@ export class ChromedashHeader extends LitElement {
     }
 
     // user is passed in from chromedash-app
-    if (this.user && this.user.email) return;
+    if (this.user !== undefined) return;
 
     // user is passed in from chromedash-app, but the user is not logged in
-    if (!this.user) {
+    if (this.user === null) {
```

Comment:
Do you really mean "=== null".  Where could the null value be coming from?  Maybe add a comment about that.